mira/metamodel/templates.py

```diff
@@ -83,7 +83,7 @@ def get_curie(self, config: Optional[Config] = None) -> Tuple[str, str]:
         for prefix in config.prefix_priority:
             identifier = self.identifiers.get(prefix)
             if identifier:
-                return prefix, identifier
+                return f"{prefix}:{identifier}"
```

Comment:
I misunderstood what you were mentioning before, this should most definitely be a tuple. The code introduced earlier that just returns a string is a problem